spec/features/subscribe_spec.rb

```diff
@@ -30,8 +30,9 @@
       visit "/email/subscriptions/new?topic_id=#{topic_id}"
       fill_in :address, with: address
       expect(page).to have_content("Test Subscriber List")
-      click_button "Subscribe"
-      expect(page).to have_content("Subscription successfully created")
+
+      page.execute_script("document.querySelector('form').submit()")
```

Comment:
Why do you prefer this over `click_button`?

Reply:
I don't, but currently `govuk_publishing_components` doesn't render the real button component in a test environment, it just puts a kind of fixture in its place that looks like this:

`<test-govuk-component data-template="govuk_component-button">{"text":"Subscribe","margin_bottom":true}</test-govuk-component>`

We didn't have this problem before because we weren't using a component. It's kind of annoying, but I think this is something the #publishing-frontend team want to improve for all components eventually.

Reply:
Ok... fair enough. Can you add a comment?